lib/fluent/supervisor.rb

```diff
@@ -1196,6 +1205,28 @@ def build_system_config(conf)
       system_config
     end
 
+    def build_additional_configuration
```

Comment:
```suggestion
    def build_additional_configurations
```

How about a name that shows there can be more than one?

Reply:
Fixed, thanks.